lib/pact_broker/client/git.rb

```diff
@@ -92,6 +95,17 @@ def self.execute_git_command
         `#{COMMAND}`
       end
 
+      def self.execute_git_commit_command(raise_error)
+        `#{COMMIT_COMMAND}`
+      rescue StandardError => e
+        if raise_error
+          raise PactBroker::Client::Error,
+                "Could not determine current git commit using command `#{COMMIT_COMMAND}`. #{e.class} #{e.message}"
+        else
+          return []
```

Comment:
why does it return an array and not nil?

Reply:
it returns nil now :)